packages/node/src/utils.ts

```diff
@@ -59,7 +59,12 @@ export function entrypointToOutputPath(
 }
 
 export function logError(error: Error) {
-  console.error(error.message);
+  let message = error.message;
+  if (!message.startsWith('Error:')) {
+    message = `Error: ${message}`;
+  }
+  console.error(message);
```

Comment:
This is an improvement, but I realized we show errors differently between startup/runtime and serverless-node/edge. We should standardize those differences, but not in this PR.